docs/content/preview/explore/change-data-capture.md

```diff
@@ -166,6 +166,135 @@ table public.projects: INSERT: project_id[integer]:1 name[character varying]:'Pr
 COMMIT 3
 ```
 
+## Try it out with LSN type HYBRID_TIME
+
+### Create a table
+
+Create a table to be streamed using `pg_recvlogical`.
+
+```sql
+CREATE TABLE test (id INT PRIMARY KEY);
+```
+
+### Create a replication slot with LSN type HYBRID_TIME
+
+Create a logical replication slot with the output plugin `test_decoding` and LSN type `HYBRID_TIME` using the following:
+
+```sql
+CREATE_REPLICATION_SLOT test_logical_replication_slot LOGICAL test_decoding HYBRID_TIME;
+```
+
+```output
+           slot_name           | consistent_point |    snapshot_name    | output_plugin
+-------------------------------+------------------+---------------------+---------------
+ test_logical_replication_slot | 0/2              | 7127147414403211264 | test_decoding
+```
+
+### Start pg_recvlogical
+
+The pg_recvlogical binary can be found under `<yugabyte-db-dir>/postgres/bin/`.
+
+Open a new shell and start pg_recvlogical to connect to the `yugabyte` database with the superuser `yugabyte` and replicate changes using the replication slot you created as follows:
+
+```sh
+./pg_recvlogical -d yugabyte \
+  -U yugabyte \
+  -h 127.0.0.1 \
+  --slot test_logical_replication_slot \
+  --start \
+  -f -
+```
+
+### Insert records
+
+Insert records and verify the replication by observing the output of `pg_recvlogical`.
+
+```sql
+INSERT INTO test VALUES (1);
+INSERT INTO test VALUES (2);
+```
+
+```output
+BEGIN 2
+table public.test: INSERT: id[integer]:1
+COMMIT 2
+BEGIN 3
+table public.test: INSERT: id[integer]:2
+COMMIT 3
+```
+
+### Kill pg_recvlogical and insert more records
+
+Kill `pg_recvlogical` using `Ctrl+C` and then insert more records.
+
+```sql
+INSERT INTO test VALUES (3);
+INSERT INTO test VALUES (4);
+INSERT INTO test VALUES (5);
+```
+
+Notice that since `pg_recvlogical` is not running, it will not be streaming the above inserted records.
+
+### Get current hybrid time LSN
+
+Use the following query to get the current hybrid time LSN.
+
+```sql
+CREATE OR REPLACE FUNCTION get_current_lsn_format()
+RETURNS text AS $$
+DECLARE
+    ht_lsn bigint;
+    formatted_lsn text;
+BEGIN
+    SELECT yb_get_current_hybrid_time_lsn() INTO ht_lsn;
```

Comment:
Is it possible to highlight only `yb_get_current_hybrid_time_lsn()` here?

Reply:
I don't think it's possible to highlight a term. Perhaps @ddhodge would know how to do that, if it's at all possible.